ds-prep-python-lists/main.py

```diff
@@ -0,0 +1,21 @@
+ages = [29 , 19 , 17 , 26 , 32 , 16 , 21]
+print(ages)
+print(ages[4])
+print(ages[-2])
+print(ages[2:6])
+print(ages[:4])
+print("17" in ages)
+print("42" in ages)
+ages[2] = "18"
+print(ages)
+ages[4] = "temp."
+ages [4] = ages[3]
+ages[3] = 32
+print(ages)
+ages.append("45")
```

Comment:
`"45"` is not the same as `45`:

---

![image](https://user-images.githubusercontent.com/16143771/149632372-4142c4cb-6e91-48b0-9591-cdd1d8a5ecc0.png)